src/managedwriter/writer_client.ts

```diff
@@ -69,6 +74,10 @@ export class WriterClient {
       connectionList: [],
     };
     this._open = false;
+    this._retrySettings = {
+      enableWriteRetries: false,
+      maxRetryAttempts: 4,
```

Comment:
Why these values? Can they be overridden?

Reply:
they can be overwritten via the `enableWriteRetries` and `setMaxRetryAttempts` methods. The default values came from the Go ManagedWriter - https://github.com/googleapis/google-cloud-go/blob/5a8c88956ea11eaf3e8aa8cffadfca06bf58c51d/bigquery/storage/managedwriter/retry.go#L31